README.md

```diff
@@ -54,6 +54,36 @@ Therefore, restic supports the following backends for storing backups natively:
 - [Google Cloud Storage](https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#google-cloud-storage)
 - And many other services via the [rclone](https://rclone.org) [Backend](https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#other-services-via-rclone)
 
+## NeoFS 
+
+As backed you can use [NeoFS](https://github.com/nspcc-dev/neofs-node).
```

Comment:
Maybe we should mention that building restuc with NeoFS plugin requires go>=1.17?